89 husky install deprecated

GMSoudersJr opened the pull request:
## Previously

![image](https://github.com/user-attachments/assets/7139eb7d-9e1a-46e1-8307-5a6c5d716de8)

## Currently

![image](https://github.com/user-attachments/assets/1e2361a9-e251-4f93-9e15-b496eb29723e)

GMSoudersJr committed:
chore(husky): remove deprecated lines

GMSoudersJr committed:
fix(husky): update README.md and script command

husky install is deprecated, the documentation also said to use
postinstall instead of prepare because yarn does not support it. Updated
the README.md to mirror the script change as well.

closes #89

ivanms1 commented on README.md:
I think you don't need `run` here, but not 100% sure.

ivanms1 commented on .husky/pre-commit:
not opposed to it, but can I know the reason for removing this?

ivanms1 reviewed (approved):
Thank you!
just a small suggestion and a question.

GMSoudersJr commented on .husky/pre-commit:
sure.

> # And there's more
> ![image](https://private-user-images.githubusercontent.com/44438849/366264584-d6fec959-9206-4eef-8db2-dbc8a1269ef4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjcyMjUxNjEsIm5iZiI6MTcyNzIyNDg2MSwicGF0aCI6Ii80NDQzODg0OS8zNjYyNjQ1ODQtZDZmZWM5NTktOTIwNi00ZWVmLThkYjItZGJjOGExMjY5ZWY0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTI1VDAwNDEwMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU2MDYxYjYzNzcxNWU0NjNmMzdmZTAzZjQ1MjMxZjc3Y2Y0YzIzODM3ZmEwOTQxNWQyYTFiYjI0NjdmMTFiYzcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.Zlgvguan8RacWz5fwZbhr4GDyluMgJUonmeYj_4xhEE)

This was another message that appeared when running husky. I originally posted this in the issue. I guess it is better to include it in the PR next time.

GMSoudersJr commented on README.md:
okay, neither am I. I was just following husky's docs.

![image](https://github.com/user-attachments/assets/8de295c4-b624-460f-96c2-aa9e2744da97)

After looking through the yarn docs:
![image](https://github.com/user-attachments/assets/cfd3fd85-57da-416f-88f9-2f2c154f68e4)

Okay, I will get rid of it.

GMSoudersJr commented:
# 🤔 Is `yarn postinstall` needed in `README.md` anymore?

## After reading more of the docs, I came across this: *post*install will run directly after install. No further commands needed.
![image](https://github.com/user-attachments/assets/7901b5cc-f395-4edc-8c9f-38b564d4c69b)


## I ran `yarn`, which equates to `yarn install` and well, here's the output:
![image](https://github.com/user-attachments/assets/9b357b98-2feb-40b6-9bda-748b4ecdbddb)

I thought it might be because I already have the project and ran husky before. So, I:
1. Created a new directory
2. Cloned this repo into it
3. Went into that directory
4. `git switch` this branch
5. `yarn` then...
![image](https://github.com/user-attachments/assets/f26bef6d-7832-4968-b6f1-d558b42a8989)

Changed some code and the linter ran as usual. 👍

drecali commented on README.md:
This seems legit to me

drecali reviewed (approved):
Looks good to me 🧶 Thanks!

GMSoudersJr committed:
Merge branch 'master' of github.com:Free-Code-Camp-Seoul/fcc-seoul-website into 89-husky-install-deprecated

GMSoudersJr committed:
docs(README.md): update README.md

remove instructions for yarn postinstall since it is automatically run
after installation.
